opts/opts.go

```diff
@@ -186,6 +188,34 @@ func ValidateAttach(val string) (string, error) {
 	return val, fmt.Errorf("valid streams are STDIN, STDOUT and STDERR.")
 }
 
+func ValidateWeightDevice(val string) (string, error) {
```

Comment:
Two things here:
- We can't do this on the client, as it's not guaranteed to be running on the same host than the daemon
- We should really use stat instead of parsing the output of `ls -l`